docs/pages/kubernetes-access/getting-started/local.mdx

```diff
@@ -0,0 +1,349 @@
+that this is working:
+
+```
+$ curl --insecure https://localhost:443/webapi/ping
+{"auth":{"type":"local","second_factor":"otp","preferred_local_mfa":"otp","has_motd":false},"proxy":{"kube":{"enabled":true,"listen_addr":"0.0.0.0:3026"},"ssh":{"listen_addr":"[::]:3023","tunnel_listen_addr":"0.0.0.0:3024","public_addr":"teleport-cluster:443"},"db":{"mysql_listen_addr":"0.0.0.0:3036"},"tls_routing_enabled":false},"server_version":"8.2.0","min_client_version":"7.0.0"}%                           
+```
+
+<Admonition type="warning" title="Certificate warning"> 
+The Teleport Proxy Service requires a TLS certificate and private key. In this
+guide, Teleport runs with a self-signed certificate. For convenience, we
+configure HTTP clients not to verify the certificate.
+
+In production setups, you will need to configure Teleport to use a certificate
+from a certificate authority like Let's Encrypt.
+</Admonition>
+
+### Deploy Kubernetes Dashboard
+
+Deploy Kubernetes Dashboard using the following command:
+
+```code
+$ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.4.0/aio/deploy/recommended.yaml
+```
+
+Verify that the dashboard and metrics scraper services are running:
+
+```code
+$ kubectl get services -n kubernetes-dashboard
+NAME                        TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+dashboard-metrics-scraper   ClusterIP   10.100.9.163   <none>        8000/TCP   8s
+kubernetes-dashboard        ClusterIP   10.100.80.65   <none>        443/TCP    8s
+```
+
+The `kubernetes-dashboard` service has an open HTTPS port but is not accessible
+outside the cluster (i.e., it has no external IP). By enabling Teleport
+Application Access, we will alow users to securely access the dashboard.
+
+<Notice type="tip"> 
+
+If installing the dashboard leads to an unexpected result,
+check the following documentation for updated installation steps:
+[Deploying the Dashboard UI](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/#deploying-the-dashboard-ui).
+
+</Notice>
+
+## Step 2/4 Create a user
+
+We will create a Teleport user who can access Kubernetes Dashboard from
+outside the cluster. To do this, we will use the `tctl` administrative tool from
+inside the pod that runs the Auth Service and Proxy Service.
+
+Run the following commands to get the name of the relevant pod and execute a
+`tctl` command to create a user:
+
+```code
+# We will use this variable for subsequent commands
+$ PROXY_POD=$(kubectl get po -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
+$ kubectl exec $PROXY_POD -- tctl users add --roles=access appuser
+User "appuser" has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h:
+https://teleport-cluster.teleport-cluster.svc.cluster.local:443/web/invite/<token>
+```
+
+Next, open a browser at `https://localhost:443/web/invite/<token>`, copying the
```

Comment:
original url (`https://teleport-cluster.teleport-cluster.svc.cluster.local:443/web/invite/<token>`) worked for me just fine, so maybe we can skip mentioning `localhost` here and just advice to visit url above?